HOW-TO-READ-ERRORS.md

```diff
@@ -12,12 +12,12 @@ error[E0277]: the trait bound ...
    |
    ...
 ```
-It means that provider cannot resolve that `SERVICE`. Possible reasons are:
+It means that the provider cannot resolve that `SERVICE`. Possible reasons are:
 1. You do not register this dependency in `ServiceProvider`.
```

Comment:
Not related to your changes, but the wording of this could be improved too. What do you think of this?
```
1. `SERVICE` was not registered as a dependency in `ServiceProvider`.
```